include/sensors/core/tasks/capacitive_sensor_callbacks.hpp

```diff
@@ -72,10 +73,15 @@ struct ReadCapacitanceCallback {
         }
         auto capacitance = convert_capacitance(measurement, number_of_reads,
                                                current_offset_pf);
-        auto message = can_messages::ReadFromSensorResponse{
-            .sensor = SensorType::capacitive,
-            .sensor_data = convert_to_fixed_point(capacitance, 15)};
-        can_client.send_can_message(can_ids::NodeId::host, message);
+        if (utils::tag_in_token(m.id.token,
+                                utils::ResponseTag::IS_THRESHOLD_SENSE)) {
+            set_threshold(capacitance + 1);
```

Comment:
Why are we adding a 1 here?

Reply:
That's the threshold. It's the current average, plus a little bit. In a followup PR I definitely want to make that `1` settable from python.